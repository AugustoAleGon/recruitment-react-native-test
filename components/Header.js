import React from 'react'
import { StyleSheet, View, Text } from 'react-native'

const Header = () => (
  <View style={styles.header}>
    <Text style={styles.headerTitle}>Acuris - mobile app</Text>
  </View>
)

const styles = StyleSheet.create({
  header: {
    flex: 1,
    maxHeight: '10%',
    backgroundColor: '#000',
    justifyContent: 'center'
  },
  headerTitle: {
    color: '#fff',
    fontSize: 20,
    paddingLeft: 10
  }
})

export default Header
