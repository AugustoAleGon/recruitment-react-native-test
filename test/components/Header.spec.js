import React from 'react'
import renderer from 'react-test-renderer'
import Header from '../../components/Header'

describe('Header', () => {
  it('should render the text of the header', () => {
    const wrapper = renderer.create(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
