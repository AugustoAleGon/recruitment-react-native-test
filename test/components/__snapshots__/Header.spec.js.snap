// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the text of the header 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#000",
      "flex": 1,
      "justifyContent": "center",
      "maxHeight": "10%",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#fff",
        "fontSize": 20,
        "paddingLeft": 10,
      }
    }
  >
    Acuris - mobile app
  </Text>
</View>
`;
